ga4gh/server/frontend.py

```diff
@@ -279,16 +279,22 @@ def configure(configFile=None, baseConfig="ProductionConfig",
     app.serverStatus = ServerStatus()
 
     app.backend = _configure_backend(app)
-    app.secret_key = os.urandom(SECRET_KEY_LENGTH)
+    if app.config.get('SECRET_KEY'):
+        app.secret_key = app.config['SECRET_KEY']
+    else:
+        app.secret_key = 'super_secret_CHANGE_ME'
+    if app.config.get('CACHE_DIRECTORY'):
+        app.cache_dir = app.config['CACHE_DIRECTORY']
+    else:
+        app.cache_dir = '/tmp/ga4gh'
+    app.cache = FileSystemCache(app.cache_dir, threshold=500, default_timeout=300, mode=384)
```

Comment:
Ya the timeout isn't too big a deal because after it times out, it can automatically request a new token from the provider. The threshold might be a little low though. I'm happy to increase both.